deployments/hadoop-yarn/ansible/20-install-spark.yml

```diff
@@ -33,7 +33,7 @@
     - name: "Download and unpack the {{spname}} tar gzip file"
       become: true
       unarchive:
-        src:  "https://downloads.apache.org/spark/{{spname}}/{{spfull}}.tgz"
+        src:  "https://archive.apache.org/dist/spark/{{spname}}/{{spfull}}.tgz"
```

Comment:
Hasn't this already been done in another PR ?

Reply:
Merging master back into the fork seems to have fixed it this time, but I think [`git rebase`](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) is the better way to fix something like this.

Reply:
Can we confirm that this is no longer part of the pull request ?
If merging master back in to this branch resolved it, then why is this still showing up in the  pull request as a change for review ?

Reply:
Same as above, I don't see this in the changed files of the PR https://github.com/wfau/aglais/pull/517/files

Reply:
OK, thanks for checking.